src/main/kotlin/org/dashevo/dpp/contract/ContractFactory.kt

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2020-present, Dash Core Team
+ *
+ * This source code is licensed under the MIT license found in the
+ * COPYING file in the root directory of this source tree.
+ */
+
+package org.dashevo.dpp.contract
+
+import org.dashevo.dpp.Factory
+import org.dashevo.dpp.util.HashUtils
+
+class ContractFactory : Factory() {
+
+    fun createDataContract(rawDataContract: MutableMap<String, Any>): Contract {
+
+        val contractId = if (rawDataContract.containsKey("contractId")) rawDataContract["contractId"] as String else ""
+
+        val contract = Contract(contractId,
+                rawDataContract["documents"] as MutableMap<String, Any>)
+
+        if (rawDataContract.containsKey("\$schema")) {
+            contract.schema = rawDataContract["\$schema"] as String
+        }
+
+        if (rawDataContract.containsKey("version")) {
```

Comment:
What version of DPP are you targeting? I ask because `version` is being removed in dpp 0.12 (https://github.com/dashevo/js-dpp/pull/128).

Reply:
Yes, this is for 0.11.